src/main/java/core/basesyntax/Array.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Array {
+    double getArray();

```

Comment:
its not array its area
```suggestion
public interface AreaCalculator {
    double getArea();
```